hooks/pre/32-install_selinux_packages.rb

```diff
@@ -2,20 +2,10 @@
 if facts.dig(:os, :selinux, :enabled)
   packages = []
 
-  if el7?
-    packages << 'rh-postgresql12-postgresql-server' if local_postgresql?
-    packages << 'rh-redis5-redis' if local_redis?
-  end
-
   packages << 'foreman-selinux' if foreman_server?
   packages << 'katello-selinux' if katello_enabled?
   packages << 'candlepin-selinux' if katello_enabled?
   packages << 'pulpcore-selinux' if pulpcore_enabled?
 
-  if katello_enabled? && el8?
```

Comment:
Perhaps this could be a separate PR and already merged. No need to wait for it IMHO.

Reply:
https://github.com/theforeman/foreman-installer/pull/786